html/about.html

```diff
@@ -10,16 +10,35 @@
   <link rel="stylesheet" href="../css/main.css">
 </head>
 
-<body class="theme-normal">
+<body class="about theme-normal">
   <nav>
-    <div id="burger" class="burger-container">
-      <span class="burger"></span>
+    <div class="sub-menu">
+      <ul class="sub-menu-social">
+        <li>
+          <a><div class="face-icon"></div></a>
+        </li>
+        <li>
+          <a><div class="twit-icon"></div></a>
+        </li>
+        <li><a class="logout" href="#">Logout</a></li>
+      </ul>
+    </div>
+    <div class="main-menu">
+      <div id="burger" class="burger-container">
+        <span class="burger"></span>
+      </div>
+      <a href="../index.html" class="event-logo" id="nav-logo"></a>
+      <ul id="nav-desktop-menu">
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="#past-events">Past events</a></li>
+        <li class="stylefy"><a href="#">Stylefy!</a></li>
+      </ul>
     </div>
   </nav>
 
   <div id="mobile-menu" class="mobile-menu">
     <ul id="mobile-menu-links">
-      <li id='stylefy'> <a href="#recent-works">Stylefy!!</a></li>
+      <li class="stylefy" id='stylefy'> <a href="#">Stylefy!!</a></li>
```

Comment:
- [ ] You left a space between the closing tag of `li` and the opening tag of the `a` elements for no reason. Let's  remove the empty space to ensure a cleaner code.

Reply:
thanks for the feedback! Will fix it right away!